lib/sprockets/erb_processor.rb

```diff
@@ -1,31 +1,32 @@
 # frozen_string_literal: true
 require 'erb'
 
-module Sprockets
-  class ERBProcessor
-    # Public: Return singleton instance with default options.
-    #
-    # Returns ERBProcessor object.
-    def self.instance
-      @instance ||= new
-    end
+class Sprockets::ERBProcessor
```

Comment:
Clever!

You might need to add a 'module Sprockets;end' before this though, in case just this file is imported